configure.ac

```diff
@@ -0,0 +1,107 @@
+AC_PREREQ(2.61)
+AC_INIT([pjdfstest],[0.1],)
+AC_CONFIG_AUX_DIR([.])
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_FILES([ \
+    Makefile \
+])
+
+AC_CANONICAL_HOST
+
+AC_PROG_CC
```

Comment:
On a FreeBSD 12 system I got compile failures because autoconf tried to use GCC instead of clang.  It seems that when FreeBSD's sys/param.h includes GCC's sys/types.h, there are some unknown type name errors.  Regardless, we should always prefer Clang on FreeBSD.  This change will do it:
```
--- a/configure.ac
+++ b/configure.ac
@@ -9,7 +9,7 @@ AC_CONFIG_FILES([ \
 
 AC_CANONICAL_HOST
 
-AC_PROG_CC
+AC_PROG_CC([cc clang gcc])
 
 # For _GNU_SOURCE on Linux, etc.
 AC_USE_SYSTEM_EXTENSIONS
```